README.md

```diff
@@ -41,12 +45,35 @@ rhsso_rhn_id: '<sso_product_id>'
 
 where `sso_product_id` is the ID for the specific Red Hat Single Sign-On version, ie. _101971_ will install version _7.5_)
 
-
-## Included roles
+### Install role
 
 * [`keycloak`](https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak/README.md): role for installing the service. _Requires: python3-netaddr_
+
+### Example installation command
+
+Execute the following command from the source root directory 
+
+```
+ansible-playbook -i ansible_hosts -e @rhn-creds.yml playbooks/keycloak.yml -e keycloak_admin_password=<changeme>`
+``` 
+
+## Configuration
+
+### Config Playbook
+
+`playbooks/keycloak-realm.yml` creates the realm, provided client(s), client role(s) and client user(s) if they don't exist.
+
+### Config role
+
 * [`keycloak_realm`](https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak_realm/README.md): role for configuring a realm, with clients and users, in an installed service.
 
+### Example configuration command
+
+Execute the following command from the source root directory
+
+```
+ansible-playbook -i ansible_hosts -e @rhn-creds.yml playbooks/keycloak.yml -e keycloak_admin_password=<changeme> -e keycloak_realm=test`
+```
 
```

Comment:
Please add a note here telling ansible_hosts is the inventory containing and keycloak instance, and add a few lines showing the example inventory for running against localhost (the file below)

Reply:
Updated the README.md accordingly commit [link](https://github.com/ansible-middleware/keycloak/pull/5/commits/c78aaece81365f0dff91ed64735e6c5ab677858d)